airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4427,12 +4479,18 @@ static void powerOnTestSensors() {
 	if (cfg::dht_read) {
 		dht.begin();										// Start DHT
 		debug_outln_info(F("Read DHT..."));
+		pcf8575.digitalWrite(P5, HIGH);
+		delay(2000);
+		pcf8575.digitalWrite(P5, LOW);
 	}
 
 	if (cfg::rtc_read) {
 		rtc.begin();
 		debug_outln_info(F("Read Time from RTC..."));
 		init_RTC();
+		pcf8575.digitalWrite(P2, HIGH);
```

Comment:
@Warukira can we also have this during power on test?

Reply:
@chegejames , excuse me if I'm getting this wrong, but it currently is in the power on test.